contextvh/src/main/java/contextvh/util/MapUtils.java

```diff
@@ -74,7 +74,7 @@ public static MultiPolygon getStakeholderLands(final Integer connectionID, final
 		MultiPolygon result = JTSUtils.EMPTY;
 		final ItemMap<Land> lands = EventManager.getItemMap(connectionID, MapLink.LANDS);
 		for (Land land : lands) {
-			if (land.getOwnerID() == stakeholderID) {
+			if (land.getOwnerID().equals(stakeholderID)) {
 				result = JTSUtils.union(result, land.getMultiPolygon());
```

Comment:
is the ID an Integer of an int, because if it is an int you may as well use == as it is the same. If it is an Integer you need to use .equals as it can be null